src/OpenTelemetry/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Modified Sampler implementation to match the
+  [spec](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/sdk.md#sampler)
+  ([#1037](https://github.com/open-telemetry/opentelemetry-dotnet/pull/1037))
```

Comment:
```suggestion
* Modified Sampler implementation to match the spec
  ([#1037](https://github.com/open-telemetry/opentelemetry-dotnet/pull/1037))
```

The PR should already have the link to the spec so we don't need it in the changelog.